### DISCUSSION TITLE
Sunbird RC Setup Failed in Local

### DISCUSSION THREAD

**Discussion by priyanka-TL** [Sunbird-Lern/Community#17] [Issues]
Like Course Certificate generation, In ML services, We have to generate improvement project certificates. You can refer to the JIRA ticket here for the improvement project certificate story 
https://project-sunbird.atlassian.net/browse/SB-28231

So we are trying to understand how it works and generate the certificate. I have followed the below guide to set up the RC
https://project-sunbird.atlassian.net/wiki/spaces/UM/pages/3085500423/Sunbird+RC+Setup

But while building the registry I got an error 
![Screenshot from 2022-06-23 10-48-03](https://user-images.githubusercontent.com/71248340/175220967-2cee8897-b6c5-492a-a106-ae043f26395f.png)

Can you please help to fix this error? I am trying to see how we can use the Sunbird RC to generate the certificate and understand how it will work for the project use case

cc - @aks30 @vijiurs @kiranharidas187 @varsha-sl @prateek-slokam @aishwaryashikshalokam

**Comment by rhwarrier** [comment]
Closing this thread since this seems to have been answered on the RC discussion forum : https://github.com/Sunbird-RC/community/discussions/236
